cmd/infracost/run.go

```diff
@@ -468,6 +469,15 @@ func (r *parallelRunner) runProjectConfig(ctx *config.ProjectContext) (*projectO
 				)
 			}
 
+			var apiErr *apiclient.APIError
+			if errors.As(err, &apiErr) {
+				if apiErr.Code != http.StatusForbidden {
```

Comment:
We also return 403/forbidden if they have a bad api key so we need to parse the message or (or maybe we can add additional context to the response) to decide if it is a quota issue.

Reply:
I was originally going to parse the message but thought that would be pretty flaky, as if we change the messaging it will break. I'll do it for now, but in the future i think we need to add a `code` or equivalent field to the hcpapi json response.